### DISCUSSION TITLE
Game keeps saying internal server error on my main but when I use my alt it works fine

### DISCUSSION THREAD

**Discussion by Haichiroku** [FabricMC/fabric-api#1691] [Player support]
Added Flans (The fabric claim plugin not the weapons mod) and made a 25k block claim on my base. I was using my alt account during this time and I transferred the claim to my main. I tried logging into my main but it keeps saying internal server error. I hope back on my alt account and the server lets me in normally.

<html>
<body>

[14:11:35] [Server thread/WARN]: Failed to handle packet for /112.201.109.118:15181
--
199 | java.lang.NullPointerException: null
200 | at io.github.flemmli97.flan.player.PlayerClaimData.read(PlayerClaimData.java:432) ~[flan-1.16.5-1.6.2-fabric.jar:?]
201 | at io.github.flemmli97.flan.event.PlayerEvents.readClaimData(PlayerEvents.java:17) ~[flan-1.16.5-1.6.2-fabric.jar:?]
202 | at net.minecraft.class_29.handler$bbf000$load(class_29.java:523) ~[intermediary-server.jar:?]
203 | at net.minecraft.class_29.method_261(class_29.java:59) ~[intermediary-server.jar:?]
204 | at net.minecraft.class_3324.method_14600(class_3324.java:344) ~[intermediary-server.jar:?]
205 | at net.minecraft.class_3324.method_14570(class_3324.java:138) ~[intermediary-server.jar:?]
206 | at net.minecraft.class_3248.method_14384(class_3248.java:113) ~[intermediary-server.jar:?]
207 | at net.minecraft.class_3248.redirect$bcl000$handlePlayerJoin(class_3248.java:562) ~[intermediary-server.jar:?]
208 | at net.minecraft.class_3248.method_18785(class_3248.java:65) ~[intermediary-server.jar:?]
209 | at net.minecraft.class_2535.method_10754(class_2535.java:228) ~[intermediary-server.jar:?]
210 | at net.minecraft.class_3242.method_14357(class_3242.java:173) ~[intermediary-server.jar:?]
211 | at net.minecraft.server.MinecraftServer.method_3813(MinecraftServer.java:882) ~[intermediary-server.jar:?]
212 | at net.minecraft.class_3176.method_3813(class_3176.java:312) ~[intermediary-server.jar:?]
213 | at net.minecraft.server.MinecraftServer.method_3748(MinecraftServer.java:811) ~[intermediary-server.jar:?]
214 | at net.minecraft.server.MinecraftServer.method_29741(MinecraftServer.java:670) ~[intermediary-server.jar:?]
215 | at net.minecraft.server.MinecraftServer.method_29739(MinecraftServer.java:257) ~[intermediary-server.jar:?]
216 | at java.lang.Thread.run(Thread.java:748) [?:1.8.0_292]
217 | [14:11:35] [Server thread/INFO]: com.mojang.authlib.GameProfile@153b6c25[id=f762d348-d61d-36f5-8614-408efeb22b94,name=Haichiroku,properties={},legacy=false] (/112.201.109.118:15181) lost connection: Internal server error

</body>
</html>

**Comment by Technici4n** [comment]
Not a fabric issue, report to flan.
